Firmware/mmu2.cpp

```diff
@@ -861,49 +819,56 @@ StepStatus MMU2::LogicStep(bool reportErrors) {
         }
     }
 
-    if( logic.Running() ){
+    if (logic.Running()) {
         state = xState::Active;
     }
     return ss;
 }
 
 void MMU2::filament_ramming() {
-    execute_extruder_sequence((const E_Step *)ramming_sequence, sizeof(ramming_sequence) / sizeof(E_Step));
+    execute_extruder_sequence(ramming_sequence, sizeof(ramming_sequence) / sizeof(E_Step));
 }
 
 void MMU2::execute_extruder_sequence(const E_Step *sequence, uint8_t steps) {
-    st_synchronize();
+    planner_synchronize();
+    Enable_E0();
+
     const E_Step *step = sequence;
     for (uint8_t i = 0; i < steps; i++) {
-        current_position[E_AXIS] += pgm_read_float(&(step->extrude));
-        plan_buffer_line_curposXYZE(pgm_read_float(&(step->feedRate)));
-        st_synchronize();
+        const float es = pgm_read_float(&(step->extrude));
+        const feedRate_t fr_mm_m = pgm_read_float(&(step->feedRate));
+        planner_set_current_position_E(planner_get_current_position_E() + es);
+        planner_line_to_current_position(MMM_TO_MMS(fr_mm_m));
```

Comment:
⚠️ All the extruder sequences are already in units mm/s. `MMM_TO_MMS()` here will make the feedrate 60 times slower which explains why I'm seeing "Load to Nozzle" operations take over 4 minutes to complete :) 

Either remove the `/ 60.F` from the extruder sequences, or remove `MMM_TO_MMS()`.

ℹ️ Removing `MMM_TO_MMS()` saves 12 bytes of flash.